src/cli.js

```diff
@@ -49,11 +48,11 @@ module.exports = async () => {
   } catch (err) {
     // If error is a SemanticReleaseError then it's an expected exception case (no release to be done, running on a PR etc..) and the cli will return with 0
     // Otherwise it's an unexpected error (configuration issue, code issue, plugin issue etc...) and the cli will return 1
-    if (err instanceof SemanticReleaseError) {
+    if (err.name === 'SemanticReleaseError') {
       logger.log(`%s ${err.message}`, err.code);
     } else {
       process.exitCode = 1;
-      logger.error(err);
+      logger.error('An error occurred while running semantic-release: %s', err);
```

Comment:
what is `%s` expanding to for an `Error` instance? `error.toString()`? As mentioned in another thread, it should include error properties

Reply:
The error itself will be printed with https://github.com/semantic-release/semantic-release/blob/include-eror-props/src/lib/logger.js#L15

The output will be:
```
[Semantic release]: An error occurred while running semantic-release: Error: error message
{ Error: error message
    at module.exports (/Users/vanduynslager/git/semantic-release/test/fixtures/plugin-error-a.js:2:17)
    at bound  (internal/util.js:227:26)
    at /Users/vanduynslager/git/semantic-release/src/lib/plugins.js:22:58
    at <anonymous> errorProperty: 'errorProperty' }
```

The `%s` is used to print `[Semantic release]: An error occurred while running semantic-release: Error: error message`

Reply:
Yeah, realised that later down. Seems redundant though to log the message twice?

Reply:
I made the change, the message shouldn't be logged twice anymore